src/dist/replication/ddl_lib/replication_ddl_client.cpp

```diff
@@ -1364,6 +1364,16 @@ error_with<duplication_modify_response> replication_ddl_client::change_dup_statu
     return call_rpc_sync(duplication_modify_rpc(std::move(req), RPC_CM_MODIFY_DUPLICATION));
 }
 
+error_with<duplication_modify_response> replication_ddl_client::update_dup_fail_mode(
+    std::string app_name, int dupid, duplication_fail_mode::type fmode)
+{
+    auto req = make_unique<duplication_modify_request>();
```

Comment:
You can validate `fmode` here to avoid all replica servers coredump, some necessary protection is reasonable.

Reply:
It's protected on shell side. Something like:

```
auto fmode = fail_mode_str == "slow" ? FAIL_SLOW : FAIL_SKIP;
```

But sure, I can add another validation here.